examples/bootstrap/outputs.tf

```diff
@@ -1,24 +1,29 @@
 output "storage_account_name" {
   description = "Name of the Azure Storage Account object used for the Bootstrap."
   value       = module.bootstrap.storage_account.name
+  sensitive   = true
```

Comment:
I've added a lot of  `sensitive` attributes. These are required by tf-0.15 and allowed by tf-0.12.29 and above.

Reply:
These are required by tf-0.15 and allowed by tf-0.12.29 and above. We want 0.15.